onnx/numpy_helper.py

```diff
@@ -3,6 +3,7 @@
 import sys
 
 import numpy as np  # type: ignore
+import numpy.typing as nptyping
```

Comment:
```suggestion
import numpy.typing as nptyping  # type: ignore
```
Doesn't have NumPy stub for mypy so simply skip it like above.